columbia-html.html

```diff
@@ -385,30 +424,30 @@ <h3 id="links">7. Links</h3>
     to see it.
   </p>
   <hr>
-  <h3 id="tables">8. Tables</h3>
+  <h3 style="color: orange" id="tables">8. Tables</h3>
   <p>
     Here's a simple table with some headings and few rows:
   </p>
   <blockquote>
     <table>
       <tr>
         <th>Heading A</th>
-        <th>Heading B</th>
+        <th style="background-color: rgb(127,255,212)">Heading B</th>
         <th>Heading C</th>
       </tr>
-      <tr>
+      <tr style="background-color:rgb(127,255,212)">
```

Comment:
ez a szin tobbszor is szerepel, kicsit egyszerubb ha kiszervezed egy stilusba es akkor nem fordulhat elo az, hogy elrontod a szinkodot (es nem lesz copy-paste az egesz)